src/functions/sendCandidateResults/framework/di/inversify.config.ts

```diff
@@ -24,7 +24,9 @@ container.bind<IConfigAdapter>(TYPES.IConfigAdapter).to(ConfigAdapter);
 
 const configAdapter: IConfigAdapter = container.get<IConfigAdapter>(TYPES.IConfigAdapter);
 
-container.bind<string>(TYPES.apiKey).toConstantValue(configAdapter.apiKey);
+configAdapter.getApiKey()
+  .then(apiKey => container.bind<string>(TYPES.apiKey).toConstantValue(apiKey))
+  .catch(err => container.bind<string>(TYPES.apiKey).toConstantValue(''));
```

Comment:
Should we not throw an error when this happens?

Reply:
Yes you are right. Shouldn't swallow the error

Reply:
logging out error if there is no api key